app/controllers/ride_mechanics_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RideMechanicsController < ApplicationController
+  def create
+    # require 'pry';binding.pry
+    RideMechanic.create!(mechanic_id: params[:mechanic_id], ride_id: params[:ride_id])
```

Comment:
Awesome! This is what I was referencing in my comment on your `mechanic_controller` update action.